eegrasp/eegrasp.py

```diff
@@ -71,27 +81,35 @@ def gaussian_kernel(self, x, sigma=0.1):
 
         Notes
         -----
-        This function is supposed to be used in the PyGSP2 module but
-        is repeated here since there is an error in the available version
-        of the toolbox (03/04/2024 dd/mm/yyyy)
+        This function is supposed to be used in the PyGSP2 module but is repeated here since there
+        is an error in the available version of the toolbox.
 
         References
         ----------
-        # D. I. Shuman, S. K. Narang, P. Frossard, A. Ortega and
-        # P. Vandergheynst, "The emerging field of signal processing on graphs:
-        # Extending high-dimensional data analysis to networks and other
-        # irregular domains," in IEEE Signal Processing Magazine, vol. 30,
-        # no. 3, pp. 83-98, May 2013, doi: 10.1109/MSP.2012.2235192.
+        * D. I. Shuman, S. K. Narang, P. Frossard, A. Ortega and P. Vandergheynst, "The emerging
+        field of signal processing on graphs: Extending high-dimensional data analysis to networks
+        and other irregular domains," in IEEE Signal Processing Magazine, vol. 30, no. 3,
+        pp. 83-98, May 2013, doi: 10.1109/MSP.2012.2235192.
         """
         return np.exp(-np.power(x, 2.) / (2.*np.power(float(sigma), 2)))
 
     def compute_distance(self, coordinates=None, method='Euclidean', normalize=True):
         """
-        Method for computing the distance.
+        Computing the distance based on electrode coordinates.
+
+        Parameters
+        ----------
+        coordinates : ndarray | None.
```

Comment:
What happens when it is None?

Reply:
If `coordinates = None` the self.coordinates will be used to compute the distance. If, in that case, `self.coordinates = None` then there is nothing to compute :)